qt/scientific_interfaces/Indirect/IndirectFitDataModel.h

```diff
@@ -23,7 +23,7 @@ using namespace MantidWidgets;
    IndirectFitDataModel - Specifies an interface for updating, querying and
    accessing the raw data in IndirectDataAnalysisTabs
 */
```

Comment:
I think you might have missed removing this file from the Indirect folder? It is already in the Inelastic/Analysis folder